node/packages/sdk/lib/instrumentation/node-process-stdout-stderr.js

```diff
@@ -19,12 +19,28 @@ module.exports.install = () => {
 
   nodeStdout.write = function (...args) {
     original.stdout.write.apply(this, args);
-    attemptParseStructuredLogAndCapture(args[0]);
+    try {
+      attemptParseStructuredLogAndCapture(args[0]);
+    } catch (error) {
+      process.nextTick(() => {
+        // Prevent crashes swalling by Node.js console
+        // see: https://github.com/nodejs/node/blob/3538e1bcde45eae60c6bdbedeb3f765dbb9714c2/lib/internal/console/constructor.js#L306-L310
+        throw error;
+      });
```

Comment:
Just out of curiosity why `process.nextTick` here?

Reply:
To escape swallowing (Console method that invokes that method will swallow and not expose error, if there's a crash here).
It's important for our tests, as otherwise we won't see if our logic crashes in unexpected places